CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Removed
```

Comment:
Not a blocker but fixed would also work here. Often we use removed for breaking changes needing a major bump.

Reply:
Sure, I don't mind changing that